dotCMS/src/test/java/com/dotcms/system/event/local/LocalSystemEventsAPITest.java

```diff
@@ -212,6 +214,52 @@ public void orphanSubscriberTest() throws DotDataException {
         Assert.assertTrue(isCalled.get());
     } // orphanSubscriberTest.
 
+    /**
+     * Given scenario: We're testing that even when no subscribers are added the asyncNotify callback gets called
+     * Expected results: We expect that the callback gets called.
+     */
+    @Test
+    public void Test_Non_Blocking_Notification_On_Event_Consumed_With_No_Subscribers() {
+        final LocalSystemEventsAPI localSystemEventsAPI = APILocator.getLocalSystemEventsAPI();
+        final AtomicBoolean isOnCompleteCalled = new AtomicBoolean(false);
+        final String message = "Async notify Event.";
+        localSystemEventsAPI.asyncNotify(new TestEventType1(message), event -> {
+            System.out.println("event:" +  event );
```

Comment:
Codacy found an issue: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5856383)